full/src/main/java/apoc/load/LoadHtml.java

```diff
@@ -34,48 +37,61 @@
+                        Elements elements = document.select(query.get(key));
+                        List<Map<String, Object>> elementsParsed = getElements(elements, config);
+                        output.put(key, elementsParsed == null ? Collections.emptyMap() : elementsParsed);
             });
 
-            return Stream.of( new MapResult(output) );
-        } catch(Exception e){
-            throw new RuntimeException("Can't read the HTML from: "+ url);
+            return Stream.of(new MapResult(output) );
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException("File not found from: " + url);
+        } catch(UnsupportedEncodingException e) {
+            throw new RuntimeException("Unsupported charset: " + charset);
+        } catch(Exception e) {
+            throw new RuntimeException("Can't read the HTML from: "+ url, e);
         }
     }
 
     private List<Map<String, Object>> getElements(Elements elements, Map<String, Object> config) {
+        boolean failSilently = Util.toBoolean(config.getOrDefault("failSilently", false));
         List<Map<String, Object>> elementList = new ArrayList<>();
 
         for (Element element : elements) {
-            Map<String, Object> result = new HashMap<>();
-            if(element.attributes().size() > 0) result.put("attributes", getAttributes(element));
-            if(!element.data().isEmpty()) result.put("data", element.data());
-            if(!element.val().isEmpty()) result.put("value", element.val());
-            if(!element.tagName().isEmpty()) result.put("tagName", element.tagName());
-
-            if ( Util.toBoolean( config.getOrDefault("children", false) ) ) {
-                if(element.hasText())  result.put("text", element.ownText());
-
-                result.put("children", getElements(element.children(), config));
-            }
-            else {
-                if(element.hasText()) result.put("text", element.text());
+            try {
+                    Map<String, Object> result = new HashMap<>();
+                    if(element.attributes().size() > 0) result.put("attributes", getAttributes(element));
+                    if(!element.data().isEmpty()) result.put("data", element.data());
+                    if(!element.val().isEmpty()) result.put("value", element.val());
+                    if(!element.tagName().isEmpty()) result.put("tagName", element.tagName());
+
+                    if (Util.toBoolean(config.getOrDefault("children", false))) {
+                        if(element.hasText()) result.put("text", element.ownText());
+
+                        result.put("children", getElements(element.children(), config));
+                    }
+                    else {
+                        if(element.hasText()) result.put("text", element.text());
+                    }
+
+                    elementList.add(result);
+            } catch (Exception e) {
+                if (failSilently) {
+                    return null;
```

Comment:
scusa non conviene fare `continue`?

Reply:
> scusa non conviene fare `continue`?

Qui al posto del `continue`, che era inutile in quanto era alla fine del loop, ho messo un `log.warn(..)` con il messaggio d'errore con il tag errato